core/app/components/pubs/CreatePubButton.tsx

```diff
@@ -1,20 +1,31 @@
-import type { CommunitiesId, StagesId } from "db/public";
```

Comment:
Before, the `CreatePubButton` required `searchParams` to be passed in, which would be passed on to the PubEditor.

Now that the PubEditor is its own page, it can access its own `searchParams` easily, so this button just opens a modal for the user to pick a pubtype, then `PubTypeFormClient.tsx` shepherds the search parameters on to the PubEditor page.